ape_foundry/provider.py

```diff
@@ -78,6 +78,10 @@ class FoundryNetworkConfig(PluginConfig):
     fork_request_timeout: int = 300
     process_attempts: int = 0
 
+    # RPC defaults
+    base_fee: int = int(10e8)  # 1 gwei
+    priority_fee: int = int(2e9)  # 2 gwei
```

Comment:
Can these be defaulted to 0?

Reply:
ya that would match hardhat. that shouldnt break anything, right?

Reply:
OK, i only ask because it was an external contributor (banteg I think) who made it not be 0 so i was afraid to change it